src/styles/blocks/header.scss

```diff
@@ -0,0 +1,43 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+  margin-bottom: 50px;
```

Comment:
you can't set margins to bem block

Reply:
Thank you! i fixed this issue and add maring property to the header tag!